### ISSUE TITLE
Need help for ML/AI optimization

### ISSUE THREAD

**Issue by swiri021** [OpenKBC/multiple_sclerosis_proj#19]
@kicheolkim It would be good if we have a new person for AI/ML support. We have knowledge about that obviously, but we need more advanced optimization techniques for ML model. What do you think? Do we have to find another person for this project?

**Comment by kicheolkim** [comment]
I agree. That's a good idea if there is someone who is interested in this project with us.
